core/modules/saver-handler.js

```diff
@@ -157,7 +157,8 @@ SaverHandler.prototype.saveWiki = function(options) {
 		return false;
 	}
 	var	variables = options.variables || {},
-		template = options.template || "$:/core/save/all",
+		template = (options.template || 
+		           this.wiki.getTiddlerText("$:/config/SaveWikiButton/Template","$:/core/save/all")).trim(),
```

Comment:
@Jermolene ... If you really want to remove the whole thing, imo these 2 lines are needed, because plugins have user facing way, to tell the core, which template to use. ... The rest of the code can go into a plugin.